internal/pkg/plugin/trello/read.go

```diff
@@ -1,17 +1,25 @@
 package trello
 
-func Read(options map[string]interface{}) (map[string]interface{}, error) {
+import (
+	"fmt"
+
+	"github.com/mitchellh/mapstructure"
 
-	var opts *Options
-	var err error
+	"github.com/merico-dev/stream/pkg/util/log"
+)
 
-	if opts, err = convertMap2Options(options); err != nil {
+func Read(options map[string]interface{}) (map[string]interface{}, error) {
+	var opts Options
+	if err := mapstructure.Decode(options, &opts); err != nil {
 		return nil, err
 	}
 
-	if err := validateOptions(opts); err != nil {
-		return nil, err
+	if errs := validate(&opts); len(errs) != 0 {
+		for _, e := range errs {
+			log.Errorf("Options error: %s.", e)
+		}
+		return nil, fmt.Errorf("opts are illegal")
```

Comment:
opts and Options are not consistent

Reply:
I write this because other plugins have been written this way before, and consistency is the main consideration here. Of course, it may be more appropriate to change to `options` here, but all plugins need to be changed together. Maybe I'll make adjustments later when I add this logic to "plugin template".